src/rpc/handlers/NFTOffers.cpp

```diff
@@ -8,29 +8,38 @@
 #include <algorithm>
 #include <rpc/RPCHelpers.h>
 
-namespace RPC {
+namespace json = boost::json;
+
+namespace ripple {
```

Comment:
Are we using this namespace just so we don't have to type out `ripple::` a bunch of times in `tag_invoke`? Feel like I'd just add `using namespace ripple;` to the beginning of tag_invoke, but I guess it probably doesn't matter, since we never call `tag_invoke` ourselves.

Reply:
no, it's because `tag_invoke` needs to be in the same namespace as the type it converts to json as per `boost::json` docs.